### ISSUE TITLE
The error of loss function

### ISSUE THREAD

**Issue by OpencvW** [yhlleo/DeepSegmentor#9]
Thank you for your code.  I met this error when I retrain deepcrack model : output with device cuda:0 and dtype Long doesn't match the desired device cuda:0 and dtype Float. Could you give me some advises?  thanks.

![图片](https://user-images.githubusercontent.com/44459025/79673316-a6a45b00-8213-11ea-9734-a019a448269c.png)

**Comment by yhlleo** [comment]
It's a problem of Pytorch Version. You can find solutions [here](https://github.com/yhlleo/DeepSegmentor/pull/7/commits/aeb1281498ac008a2cb5394451f53d9594c2da6c).

**Comment by OpencvW** [comment]
Thank you for your help! the problem is solved.
